cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -95,6 +75,7 @@ import SkeletonLoader from '../components/SkeletonLoader/SkeletonLoader';
 import credentialDisplayData from '@/config/credentialDisplayData.json';
 
 const filterBy = ['IssuerManifestCredential', 'GovernanceCredential'];
+const images = require.context('@/assets/img/', false, /\.png$|\.jpg$|\.svg$/);
```

Comment:
We should lazy-load images preferably, instead of bulk-loading them

Reply:
#974 is created